packages/components/src/responsive-wrapper/style.scss

```diff
@@ -15,4 +15,5 @@
 	left: 0;
 	width: 100%;
 	height: 100%;
+	margin: auto;
```

Comment:
I guess this centers the small images, I'm not sure it's better than the left aligned images design-wise. That said, I don't have a strong preference.

Reply:
cc: @mapk to double-check if the design looks ok.